int64 a
int64 b
---#要請と応答メッセージを区分
int64 result